TorqueDataConnect/extension.json

```diff
@@ -43,7 +43,8 @@
     "torquedataconnectuploadattachment": "TorqueDataConnectUploadAttachment"
   },
   "ExtensionMessagesFiles": {
-		"TorqueDataConnectMagic": "TorqueDataConnect.i18n.php"
+		"TorqueDataConnectMagic": "TorqueDataConnect.i18n.php",
```

Comment:
Looks like this line replaces spaces with a tab character.  Can you change it back to use spaces?

Reply:
+1  (see also related discussion in [ActivityLog issue #3](https://github.com/OpenTechStrategies/ActivityLog/issues/3), which refers to a probably-soon-to-be-in-effect `CONTRIBUTING.md` file that documents this convention of spaces-not-tabs).